src/services/swap.ts

```diff
@@ -53,7 +57,7 @@ export default class SwapService {
     const parsedAmount = parseAmount(amountIn, currencyIn)
     const pairs = await this.pairService.getPairs(currencyIn, currencyOut)
 
-    if (!currencyIn || !currencyOut || !parsedAmount || !pairs) return
+    if (!currencyIn || !currencyOut || !parsedAmount || !pairs) throw new NoPoolLiquidityError()
```

Comment:
Can you give an example of the error response? Would be cool to add an error example in the docs